File: mbrs/src/main/resources/static/js/services/GradService.js
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Grad

angular.module('gremoNaElektrikoApp.GradService', [])
	.factory('GradService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/grad');
		}
		
		factory.createGrad = function(grad ) {
			return $http.post('/gremoNaElektriko/grad', grad);
		}

		factory.deleteGrad = function(id) {
			return $http.delete('/gremoNaElektriko/grad/'+id);
		}
		
		factory.updateGrad = function(grad) {
			return $http.put('/gremoNaElektriko/grad', grad);
		}
		
		return factory;
	});

File: mbrs/src/main/java/com/ftn/mbrs/service/GradService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Grad


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Grad;

public interface GradService {

	public Grad save(Grad grad);
	
	public Grad update(Grad grad);

	public String delete(Long id);
	
	public Grad findOne(Long id);
	
	public List<Grad> findAll();
}
